spec/moonshot/commands/build_spec.rb

```diff
@@ -0,0 +1,19 @@
+describe Moonshot::Commands::Build do
+
+  it 'should not handle --skip-ci-status and build correctly' do
+    soso = Moonshot::CommandLineDispatcher.new('build', subject, {})
+    op = soso.send(:build_parser, subject)
```

Comment:
Same for `op`.

Reply:
Fixed too ,, P.S. this variable name used in all commands spec tests .. may be refactoring should be applied later